internal/chart/helm_client.go

```diff
@@ -51,9 +61,75 @@ func getActionConfig(namespace string) (*action.Configuration, error) {
 	return actionConfig, nil
 }
 
-// Option to perform additional configuration of action.Install before running a chart installation.
+// InstallOption to perform additional configuration of action.Install before running a chart installation.
 type InstallOption func(install *action.Install)
 
+type postRender struct {
+	client.Client
+	namespace string
+}
+
+func (p *postRender) Run(renderedManifests *bytes.Buffer) (modifiedManifests *bytes.Buffer, err error) {
+
+	var configMapList v1.ConfigMapList
+	if err := p.Client.List(context.Background(), &configMapList); err != nil {
```

Comment:
How about getting configmaps from the namespace we are interested in? 
```golang

	var configMapList v1.ConfigMapList
	opts := &client.ListOptions{Namespace: p.namespace}
	if err := p.Client.List(context.Background(), &configMapList, opts); err != nil {
		return nil, err
	}

```

Reply:
Good idea, added.